README.md

```diff
@@ -1,28 +1,87 @@
-# tf-aws-iam-cloudability
```

Comment:
This first line represents the name of the module I would keep it also with the description.

Reply:
Thanks for the update, done.